EvidenceApi.Tests/V1/UseCase/GetEvidenceRequestsUseCaseTests.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using AutoFixture;
+using EvidenceApi.V1.Boundary.Request;
+using EvidenceApi.V1.Boundary.Response;
+using EvidenceApi.V1.Boundary.Response.Exceptions;
+using EvidenceApi.V1.Domain;
+using EvidenceApi.V1.Gateways.Interfaces;
+using EvidenceApi.V1.UseCase;
+using EvidenceApi.V1.UseCase.Interfaces;
+using FluentAssertions;
+using FluentValidation.Results;
+using Moq;
+using NUnit.Framework;
+using EvidenceApi.V1.Factories;
+
+namespace EvidenceApi.Tests.V1.UseCase
+{
+    public class GetEvidenceRequestsUseCaseTests
```

Comment:
You're not currently testing that the params with which the use case calls the gateway with are correct.

You either need to specify the specific params (or functions to match the params with) in your `Setup` call, and then call `VerifyAll` OR you can call `.Verify(x => ...)` in a test case and specify the specific params there.